main.go

```diff
@@ -45,6 +45,20 @@ var DBInstanceClassMemory = map[string]string{
 	"db.r5.24xlarge": "824633720832",
 }
 
+// DBInstanceClassConnections maps DB instance types with their max connections
+var DBInstanceClassConnections = map[string]string{
```

Comment:
is there an API in AWS sdk which gives these numbers?

Reply:
Nope unfortunately :/